packages/react-ui-core/src/Menu/MenuWrapper.js

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react'
```

Comment:
Maybe I'm missing something, but it seems like this component is unnecessary. The only addition here is `onSelect`. However, in `DropDownMenu` lines 72-73, you aren't passing a `onSelect` prop, so it is never going to be fired anyway.

Reply:
So, its not obvious, but take a look at the `Dropdown` component. It actually clones its children and passes an `onSelect` into it.